api/lib/config.js

```diff
@@ -28,14 +32,18 @@ var configPath = nconf.get('config')
 || process.env['TEST_CONFIG']
 || path.join(process.cwd(), 'config.json');
 
+if (!fs.existsSync(configPath)) {
```

Comment:
Prior to this commit one could run the tests without a config file. This is implicitly forcing either `config.json` to have been copied from `config-example.json` or --config to be specified. It's desired to be able to run the tests without a config file